dead.py

```diff
@@ -83,7 +83,15 @@ def visit_Name(self, node: ast.Name) -> None:
     def visit_Attribute(self, node: ast.Attribute) -> None:
         if isinstance(node.ctx, ast.Load):
             self.reads_target[node.attr].add(self.filename)
+        self.generic_visit(node)
 
+    def visit_Call(self, node: ast.Call) ->None:
+        """Mark the attributes that read by getattr"""
+        if (isinstance(node.func, ast.Name)
+                and node.func.id == 'getattr'
+                and isinstance(node.args[1], ast.Str)):
```

Comment:
add-trailing-comma should be fixing this but I haven't finished my branch for it, please indent as this:

```python
if (
        isinstance(node.func, ast.Name) and
        node.func.id == 'getattr' and
        isinstance(node.args[1], ast.Str)
):
    # code
```

- double indented inside condition to differentiate from code
- operators at end so conditions line up (yeah yeah, pep8 has switched on this and black disagrees but I don't care)